package.json

```diff
@@ -50,5 +52,13 @@
     "rollup-plugin-postcss": "^4.0.2",
     "ts-jest": "^29.2.5",
     "typescript": "^5.7.2"
```

Comment:
there needs to be a comma after the version specification other wise the file cannot be parsed.